base/utferror.jl

```diff
@@ -0,0 +1,30 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+##\brief      Error messages for Unicode / UTF support
```

Comment:
@tkelman and I've been spending more time than I'd like changing things so frequently :grinning: but nobody has been holding a gun to either of our heads to make you review, or me to respond to your reviews.  A long process yes, and I'm sorry for the time it has taken, but I think in the end, the final result for julia is much better.  Thanks again for your time.